alchemy/src/cloudflare/do-state-store/example.ts

```diff
@@ -0,0 +1,146 @@
+import { alchemy } from "../../alchemy.js";
+import { DOFSStateStore } from "./store.js";
+
+/**
+ * Example demonstrating DOFS state store with auto-deployment
+ * 
+ * This example shows how the DOFS state store automatically deploys
+ * a Cloudflare worker with Durable Objects when no URL is provided.
+ * 
+ * Prerequisites:
+ * - Run `wrangler login` (if not already done)
+ * - That's it! No API tokens needed.
+ */
+
+console.log("🚀 Starting DOFS State Store Auto-Deployment Example");
+console.log("=".repeat(60));
+
+try {
+  console.log("🔧 Initializing Alchemy with DOFS state store...");
+  
+  // Create Alchemy app with auto-deploying DOFS state store
+  const app = await alchemy("dofs-test-app", {
+    stage: "dev", 
+    phase: "up",
+    
```

Comment:
Ultumately the goal is to move all the tests to DOFSStateStore, so we don't need this example. Use the current examples and move them to it. Good way to test actual usage too.